docs/rules/jsx-sort-props.md

```diff
@@ -75,6 +79,36 @@ When `true`, alphabetical order is not enforced:
 <Hello tel={5555555} name="John" />
 ```
 
+### `reservedFirst`
+
+This can be a boolean or an object option.
+
+When `reservedFirst` is defined, React reserved props (`children`, `dangerouslySetInnerHTML`, `key`, `ref`) must be listed before all other props, but still respecting the alphabetical order:
```

Comment:
It does nothing on a composite component; what I mean is, it's a normal (non-reserved) prop on them.